addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2630,6 +2630,29 @@ def test_out_invoice_switch_out_refund_2(self):
             'amount_untaxed' : self.move_vals['amount_untaxed'],
         })
 
+    def test_out_invoice_switch_out_refund_3(self):
+        # Test creating an account_move with an out_invoice_type and switch it in an out_refund and a fiscal position.
+        move = self.env['account.move'].create({
+            'move_type': 'out_invoice',
+            'partner_id': self.partner_a.id,
+            'invoice_date': fields.Date.from_string('2019-01-01'),
+            'currency_id': self.currency_data['currency'].id,
+            'invoice_payment_term_id': self.pay_terms_a.id,
+            'fiscal_position_id' : self.fiscal_pos_a.id,
+            'invoice_line_ids': [
+                Command.create({
+                    'product_id': self.product_line_vals_1['product_id'],
+                    'product_uom_id': self.product_line_vals_1['product_uom_id'],
+                    'price_unit': self.product_line_vals_1['price_unit'],
+                    'tax_ids': [Command.set(self.product_line_vals_1['tax_ids'])],
+                }),
+            ],
+        })
```

Comment:
Maybe we could use the helper, since we only test the fiscal position?

```suggestion
        move = self.init_invoice('out_invoice')
        move.fiscal_position_id = self.fiscal_pos_a
```